### ISSUE TITLE
Missed GTK window (min-max-close) control icons/buttons

### ISSUE THREAD

**Issue by AmirAlfallal** [microsoft/vscode#234430]
Type: <b>Feature Request</b>

Window close, minimize and maximize buttons (icon shape) with the new look in VSCode Insider latest update on ubuntu24 - automatically changed from GTK window control icons appearance to windows system looking ones.

VS Code version: Code - Insiders 1.96.0-insider (69acde7458f428f0e6869de8915c9dd995cdda1a, 2024-11-21T05:04:38.064Z)
OS version: Linux x64 6.8.0-49-generic
Modes:

![Image](https://github.com/user-attachments/assets/cdadf498-97bf-4064-8149-6edc946f9eb8)

**Comment by benibenj** [comment]
Could you attach an image of the window control buttons? You should not be seeing the windows WCO but the ones for Linux.

**Comment by AmirAlfallal** [comment]
Here you are @benibenj 
![Image](https://github.com/user-attachments/assets/fc2aceb5-88c0-4cb8-a672-196b93565b87)

**Comment by bpasero** [comment]
@codebytere fyi, do we have control over how WCO is rendered and why it would look different here?

**Comment by deepak1556** [comment]
Yes we have control on the rendering, but the implementation today is borrowed from Chrome and it is specific to their UI. We did not use GTK api to draw the controls for reasons mentioned in https://github.com/electron/electron/pull/41840#issuecomment-2081221596. PWA also share the same implementation, unless there is a strong reason to diverge from upstream I doubt this will change.
